AsyncDisplayKit/ASCollectionView.h

```diff
@@ -171,6 +171,13 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)relayoutItems;
 
 /**
+ * Triggers a recalulation of leadingScreensForBatching
+ * @discussion During various layout changes that trigger content size changes, batch fetching needs to recalulate it's number of screens left to scroll before the delegate -collectionView:beginBatchFetchingWithContext: is called. Call when a layout transition is complete, or it will have no effect.
+ *
+ */
+- (void)invalidateBatchFetchMeasurements;
```

Comment:
@appleguy I looked really closely to try to find a way to trigger this without exposing it. The problem is, the batch invalidation is purely scroll related, and view re-layout when item size changes is not. 
Also having the control to fire this manually gave me the ability to be sure it fired once and only once at the exact end of my interactive ASDisplayNode layout transition. 
There are nuances to transitions and their completion, like during a "pinch to zoom" you let go, you must call `collectionNode.view.cancelInteractiveTransition()` or `collectionNode.view.finishInteractiveTransition()` typically depending on the transition progress. This means you would only want to fire this in the completion block of `startInteractiveTransitionToCollectionViewLayout`, which might me really tricky to detect within ASDK. 

Happy to try another approach if you have any ideas.